agentops/client.py

```diff
@@ -1,8 +1,8 @@
 """
-    AgentOps client module that provides a client class with public interfaces and configuration.
+AgentOps client module that provides a client class with public interfaces and configuration.
 
-    Classes:
-        Client: Provides methods to interact with the AgentOps service.
+Classes:
+    Client: Provides methods to interact with the AgentOps service.
 """
```

Comment:
Can you please update the docstring to show that it returns the token cost as a Decimal?

Also, update the type hint

```
def end_session(...) -> Decimal:
    ...
```

Reply:
Have added type hints.

Do you want me to add all methods of client to the docstring in the beginning of client.py?

Reply:
No need, thanks